### PR TITLE
Restores the sbt plugin release and fixes ci release

### PR DESCRIPTION
**Pull request by fedefernandez**

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.17.1"
+version in ThisBuild := "0.17.1-SNAPSHOT"
```
Comment (juanpedromoreno) [inline version.sbt]:
```suggestion
version in ThisBuild := "0.17.2-SNAPSHOT"
```

File: .travis.yml [hunk 2]
```diff
@@ -58,7 +58,9 @@ jobs:
               git clean -f;
               git checkout master;
               git pull origin master;
+              sbt ";project idlgen-sbt;release";
               sbt release;
               sbt docs/publishMicrosite;
             fi
+            if [ $TRAVIS_SCALA_VERSION = "2.12.8" ]; then sbt ";project legacy-avro-decimal-compat-protocol;release"; fi
```
Comment (juanpedromoreno) [inline .travis.yml]: Is this needed? Aren't we already in the 2.12.8 branch? https://github.com/higherkindness/mu/pull/556/files#diff-354f30a63fb0907d4ad57269548329e3R48

Comment (fedefernandez) [reply .travis.yml]: Oh, that's true

Comment (juanpedromoreno) [reply .travis.yml]: Probably `crossScalaVersions := Seq("2.12.8")` should be fine for that specific module

**Pull request merged**